{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"TimeoutError: Timed out waiting for Juju model to be ready!","children":[{"name":"test_given_sdcore_terraform_module_when_deploy_then_status_is_active","uid":"fac7d1f428bf88e7","parentUid":"fbcc66ecd3527200d23607b258c5237d","status":"broken","time":{"start":1739327185183,"stop":1739328492555,"duration":1307372},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":["asyncio","abort_on_fail"]}],"uid":"fbcc66ecd3527200d23607b258c5237d"},{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='grafana-0.grafana-endpoints.cos-lite.svc.cluster.local', port=3000): Max retries exceeded with url: /api/search?query=5G%20Network%20Overview (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x74c896eb12d0>: Failed to resolve 'grafana-0.grafana-endpoints.cos-lite.svc.cluster.local' ([Errno -3] Temporary failure in name resolution)\"))","children":[{"name":"test_given_cos_lite_integrated_with_sdcore_when_searching_for_5g_network_overview_dashboard_in_grafana_then_dashboard_exists","uid":"4102b4cb01476ca4","parentUid":"5a3e57478c06d9f9b47bee52634fbf29","status":"broken","time":{"start":1739328785562,"stop":1739328788569,"duration":3007},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":["asyncio","abort_on_fail"]}],"uid":"5a3e57478c06d9f9b47bee52634fbf29"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}